.gitmodules

```diff
@@ -15,4 +15,4 @@
 	url = https://github.com/gabime/spdlog.git
 [submodule "3rd/json"]
 	path = 3rd/json
-	url = https://github.com/nlohmann/json.git
+	url = https://github.com/nlohmann/json.git
```

Comment:
is it possible to not have this file show up as changed at all?

Reply:
i have no clue, was looking at it too weirdly, quirk of git